corehq/util/string.py

```diff
@@ -0,0 +1,18 @@
+def slash_join(*strings):
```

Comment:
You can use [urllib.parse.urljoin](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin)?

Reply:
You're absolutely right! How did I miss this before?! I'll swap out `slash_join()`. Thank you!